force-app/main/triggers/classes/HOT_WorkOrderLineItemHandler.cls

```diff
@@ -158,4 +171,11 @@ public without sharing class HOT_WorkOrderLineItemHandler extends MyTriggers {
             }
         }
     }
+    
+    public static void setDuration(List<workOrderLineItem> workOrderLineItems) {
```

Comment:
Finnes det en test-metode for denne?

Reply:
Ja, den testes i workOrderLineItemTest 127-161 (HOT_ProcessBuilderMigrationTest)